### ISSUE TITLE
Can't add a comment, get an error

### ISSUE THREAD

**Issue by 4l1fe** [utterance/utterances#509]
I saved **Console** log i get when i'm trying to post a comment. in short i get 404 and then 410 errors.
Check it out here https://4l1fe.github.io/articles/2021-02-24/
```
api.github.com/repos/4l1fe/4l1fe.github.io/contents/utterances.json?ref=master:1 Failed to load resource: the server responded with a status of 404 ()
utterances.3c7bd773.js:1 Fetch finished loading: POST "https://api.github.com/markdown".
k @ utterances.3c7bd773.js:1
fa @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
setTimeout (async)
handleInput @ utterances.3c7bd773.js:1
utterances.3c7bd773.js:1 Fetch finished loading: POST "https://api.github.com/markdown".
k @ utterances.3c7bd773.js:1
fa @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
setTimeout (async)
handleInput @ utterances.3c7bd773.js:1
utterances.3c7bd773.js:1 Fetch finished loading: POST "https://api.github.com/markdown".
k @ utterances.3c7bd773.js:1
fa @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
setTimeout (async)
handleInput @ utterances.3c7bd773.js:1
utterances.3c7bd773.js:1 POST https://api.utteranc.es/repos/4l1fe/4l1fe.github.io/issues?label=comment 410
ca @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
i @ utterances.3c7bd773.js:1
Promise.then (async)
a @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
H @ utterances.3c7bd773.js:1
o @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
(anonymous) @ utterances.3c7bd773.js:1
Na @ utterances.3c7bd773.js:1
handleSubmit @ utterances.3c7bd773.js:1

```

**Comment by koraylinux** [comment]
I checked your website's repo, it's public but issues section is not active so make it active and try again.

**Comment by 4l1fe** [comment]
> I checked your website's repo, it's public but issues section is not active so make it active and try again.

That works. Thank you much, i wouldn't guess it.

**Comment by solson** [comment]
I ran into the exact same problem on someone's blog and it has a *really* nasty consequence: utteranc.es submitted my long GitHub comment, silently failed, and cleared the form. I tried using the devtools and `about:cache` to recover, but failed, so I had to rewrite the entire long comment from scratch (and had to post it elsewhere, of course, but that part is up to the blog to solve). I only noticed while using the devtools that it HTTP 410'd, which is how I found this issue.

It would be a really good idea if utteranc.es figured out how to fail more gracefully and preserve the form data.
